src/core/include/openvino/op/adaptive_avg_pool.hpp

```diff
@@ -15,7 +15,7 @@ namespace v8 {
 /// \ingroup ov_ops_cpp_api
 class OPENVINO_API AdaptiveAvgPool : public Op {
 public:
-    OPENVINO_OP("AdaptiveAvgPool", "opset8");
+    OPENVINO_OP("AdaptiveAvgPool", "opset8", Op);
```

Comment:
Does the fix require adding the "Op" to each `OPENVINO_OP` macro? 
The `ov::op::Op` is the default for `OPENVINO_OP` macro defined here:
https://github.com/openvinotoolkit/openvino/blob/571e98d5880a30e9d8ca25f445c343e955e79123/src/core/include/openvino/op/op.hpp#L14-L21

Reply:
It's not required - reverted.